.chloggen/includemetadataotlpreceivers.yaml

```
@@ -0,0 +1,14 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
```

```suggestion
change_type: breaking
```
I think we should mark this breaking and update [this](https://github.com/signalfx/splunk-otel-collector-chart/blob/main/docs/advanced-configuration.md?plain=1#L933) section in the docs - add a note that it is by default enabled in gateway and to disable it, make the config option false.

---

I've marked as breaking and added a note on how to revert if necessary. Let me know if it could be made more clear 👍